spec.bs

```diff
@@ -2877,6 +2888,35 @@ CORP violation report=] algorithm, as leaving it unfenced may cause a privacy le
       1. Set |url| to |config|'s [=fenced frame config/mapped url=]'s [=mapped url/value=].
 
       1. Run steps in |config|'s [=fenced frame config/on navigate callback=].
+  
+    1. If |url| is a [=url=], |navigable| is a [=fenced navigable container/fenced navigable=], and
```

Comment:
Actually, I think `|url|` is _always_ a URL here, right? URN UUIDs are URLs themselves, so I think this portion of the condition is redundant?

Reply:
`|url|` can be a urn:uuid if `|navigable|` is not a fenced navigable, but that will get caught here anyway, so it should be fine to remove.